templates/constraints/yml/templates/csharp/non-sso-tab/teamsapp.local.yml.tpl.mustache

```diff
@@ -2,11 +2,41 @@
 
 provision:
 {{#script}} TAB, DOTNET {{/script}}
+
 {{#teamsAppCreate}} {{/teamsAppCreate}}
 
 {{#teamsAppValidateManifest}} {{/teamsAppValidateManifest}}
+
 {{#teamsAppZipAppPackage}} {{/teamsAppZipAppPackage}}
+
 {{#teamsAppValidateAppPackage}} {{/teamsAppValidateAppPackage}}
+
 {{#teamsAppUpdate}} {{/teamsAppUpdate}}
 
-{{#fileCreateOrUpdateJsonFile}} {"launchSettings": true, "applicationUrl": "\"https://localhost:44302;http://localhost:2544\"", "hotReload": true} {{/fileCreateOrUpdateJsonFile}}
+{{#teamsAppExtendToM365}} {{/teamsAppExtendToM365}}
+
+{{#fileCreateOrUpdateJsonFile}}
+{
+  "launchSettings": true,
+  "profiles": [
```

Comment:
If this is about supporting both metaOS and Teams, it looks like it is common across most templates. How do we define it in TS/JS templates and can it also be a contraint? @Yimin-Jin @hund030

Reply:
We can update the fileCreateOrUpdateJsonFile action's constraint, so that we can reduce duplicates in template's constraints.